scripts/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Use `"extends": "../tsconfig.json"` to have this tsconfig inherit common properties with the rest of the project.

Reply:
Resolved in next iteration